assets/Documenter.css

```diff
@@ -5,3 +5,16 @@ div.wy-menu-vertical ul.current li.toctree-l3 a {
 a.documenter-source {
   float: right;
 }
+
+/* .methodtable */
+.methodtable pre {
```

Comment:
Should these classes be prefixed with a `documenter-` just to make sure they stay separate from user-defined things?